web/docs/auth/ui.md

```diff
@@ -1,3 +1,622 @@
+  colors: {
+    brand: '#5969b8', // blue
+    brandAccent: '#de5998', // pink
+    submitButtonText: 'white',
+  },
+}
+```
+
+```jsx title="client/LoginPage.jsx"
+import { LoginForm } from '@wasp/auth/forms/Login'
+import { authAppearance } from './appearance'
+
+export function LoginPage() {
+  return (
+    <LoginForm
+      // Pass the appearance object to the form
+      appearance={authAppearance}
+    />
+  )
+}
+```
+</TabItem>
+<TabItem value="ts" label="TypeScript">
+
+```ts title="client/appearance.ts"
+import type { CustomizationOptions } from '@wasp/auth/forms/types'
+
+export const authAppearance: CustomizationOptions['appearance'] = {
+  colors: {
+    brand: '#5969b8', // blue
+    brandAccent: '#de5998', // pink
+    submitButtonText: 'white',
+  },
+}
+```
+
+```tsx title="client/LoginPage.tsx"
+import { LoginForm } from '@wasp/auth/forms/Login'
+import { authAppearance } from './appearance'
+
+export function LoginPage() {
+  return (
+    <LoginForm
+      // Pass the appearance object to the form
+      appearance={authAppearance}
+    />
+  )
+}
+```
+</TabItem>
+</Tabs>
+
+We recommend defining your appearance in a separate file and importing it into your components.
+
+### 2. Using your logo
+
+You can add your logo to the Auth UI by passing the `logo` prop to any of the components.
+
+<Tabs groupId="js-ts">
+<TabItem value="js" label="JavaScript">
+
+```tsx title="client/LoginPage.jsx"
+import { LoginForm } from '@wasp/auth/forms/Login'
+import Logo from './logo.png'
+
+export function LoginPage() {
+  return (
+    <LoginForm
+      // Pass in the path to your logo
+      logo={Logo}
+    />
+  )
+}
+```
+</TabItem>
+<TabItem value="ts" label="TypeScript">
+
+```tsx title="client/LoginPage.tsx"
+import { LoginForm } from '@wasp/auth/forms/Login'
+import Logo from './logo.png'
+
+export function LoginPage() {
+  return (
+    <LoginForm
+      // Pass in the path to your logo
+      logo={Logo}
+    />
+  )
+}
+```
+</TabItem>
+</Tabs>
+
+### 3. Social buttons layout
+
+You can change the layout of the social buttons by passing the `socialLayout` prop to any of the components. It can be either `vertical` or `horizontal` (default).
+
+If we pass in `vertical`:
+
+<Tabs groupId="js-ts">
+<TabItem value="js" label="JavaScript">
+
+```tsx title="client/LoginPage.jsx"
+import { LoginForm } from '@wasp/auth/forms/Login'
+
+export function LoginPage() {
+  return (
+    <LoginForm
+      // Pass in the socialLayout prop
+      socialLayout="vertical"
+    />
+  )
+}
+```
+</TabItem>
+<TabItem value="ts" label="TypeScript">
+
+```tsx title="client/LoginPage.tsx"
+import { LoginForm } from '@wasp/auth/forms/Login'
+
+export function LoginPage() {
+  return (
+    <LoginForm
+      // Pass in the socialLayout prop
+      socialLayout="vertical"
+    />
+  )
+}
+```
+</TabItem>
+</Tabs>
+
+We get this:
+
+![Vertical social buttons](/img/authui/vertical_social_buttons.png)
+
+### Let's put everything together 🪄
```

Comment:
Should this header start with a number like the other subheadings in this section?

Reply:
It's not part of the list of possible customisation options, that's why I didn't put the number.